file-formats/edcc/examples/z_aff_example_edcc.edcc.json

```diff
@@ -0,0 +1,136 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "example consistency scenario",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "cloudDevelopment"
+  },
+  "country": "es",
+  "taxAuthorityMessages": [
+    {
+      "messageType": "ES_INV_SUPPLIER_PULL",
+      "messageTypeDescription": "Example Spain Invoice pull",
+      "taxAuthorityDocumentType": "ES_INV_DOC"
+    }
+  ],
+  "taxAuthorityTables": [
+    {
+      "tableName": "EDCEXAMPINV"
+    }
+  ],
+  "comparisonTypes": [
+    {
+      "comparisonType": "TaxAuthVeDoc",
+      "description": "Compare Tax Authority Invoice with eDocument",
+      "positionNumber": "1",
+      "className": "CL_EDC_EXAMPLE_CLASS",
+      "dataSource": "I_EDC_EXAMPLE",
+      "existenceCheck": {
+        "checkId": "Existence",
+        "description": "Example",
+        "fieldName": "EDCEXIST",
+        "fieldValue": "X",
+        "resultProcess": "ZMISSING1"
+      },
+      "statusCheck": [
+        {
+          "checkId": "EDCSTAT",
+          "description": "Example Status Check",
+          "sequenceNumber": "1",
+          "sourceFieldname1": "InvoiceReportingStatus",
```

Comment:
this example json needs to be generated again. These properties still have the old names